build/build.py

```diff
@@ -185,30 +185,31 @@ def get_bazel_paths(bazel_path_flag):
 
 def get_bazel_path(bazel_path_flag):
   """Returns the path to a Bazel binary, downloading Bazel if not found. Also,
-  it checks Bazel's version at lease newer than 2.0.0.
+  checks Bazel's version is at least newer than 3.7.2.
```

Comment:
If .bazelversion is checking for 4.1.0, can we make it the same?

Having 2 checks that are different is confusing.

Reply:
Sure, why not. Done.

(I chose 3.7.2 since that is probably an acceptable version on most platforms, but you're probably right that picking one version is fine.)